.circleci/config.yml

```diff
@@ -495,6 +508,17 @@ workflows:
               cniNetworkPlugin: [ calico ]
           parallelism: 1
           requires: [ build-<< matrix.arch >>, check ]
+      - e2e:
+          name: << matrix.target >>:<< matrix.arch >>-<< matrix.k8sVersion >>-ebpf
+          matrix:
+            alias: test/e2e-legacy-ebpf
+            parameters:
+              k8sVersion: [ kind ]
+              target: [ test/e2e ]
+              arch: [ amd64 ]
+              ebpf: [ enabled ]
+          parallelism: 1
```

Comment:
will this run all the E2E tests but with eBPF or just ebpf test?

Reply:
This will run only this test, other tests that are going to have the label `kind-only`. We are running kind only with IPv6 and skipping for others. Unfortunately, eBPF tests are not working with k3d yet.